### PR TITLE
Add  support for Python 3 and Plone 5.2

### PR DESCRIPTION
**Pull request by pbauer**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ This product was developed for use with Plone and Dexterity.
 Requirements
 ------------
 
-* Plone >= 4.3
+* Versions >= 1.3 are for Plone 5+, if you use Plone 4.3, use versions 1.2.x
```
Comment (idgserpro) [inline README.rst]: It would be nice to have a
```
Products.CMFPlone >=5.0
```
in install_requires.

Comment (pbauer) [reply README.rst]: done

File: CHANGES.rst [hunk 2]
```diff
@@ -4,12 +4,28 @@ Changelog
 1.3.1 (unreleased)
 ------------------
 
+- Add support for Python 3 and Plone 5.2.
+  [pbauer]
+
+- Drop support for Plone 4.
+  [pbauer]
+
+- Use Ressource-Registry (Pat-Registry), Update JS/CSS, Add Uninstall
```
Comment (idgserpro) [inline CHANGES.rst]: Isn't this a small typo? `Ressource`?

File: README.rst (hunk 1 above)
Comment (idgserpro) [inline README.rst]: Should this be added to 1.2x branch as well? https://github.com/collective/collective.z3cform.datagridfield/blob/9b915e3b524672c2abbda34b4203670a46e74ffd/setup.py#L26 `Products.CMFPlone < 5`?

Comment (pbauer) [reply README.rst]: You're welcome to do that

**Comment by jensens** [comment]
Attention: Current Travis CI configuration does not test on Python 3.6./3.7!

**Pull request merged**